### ISSUE TITLE
reduction="none" problems

### ISSUE THREAD

**Issue by Zhudongsheng75** [mlpc-ucsd/BLIVA#8]
`outputs = self.llm_model(
                    inputs_embeds=inputs_embeds,
                    attention_mask=attention_mask,
                    return_dict=True,
                    labels=this_targets,
                    reduction="none",
                )
`
I noticed that the code in bliva_vicuna7b.py (locate in **_predict_class**) maybe has some problems. My transformers version is 4.28.0 and there is no reduction parameter in LlamaForCausalLM Class. In my opinion, the **_predict_class** used to evaluate the vqa task such as visdual. However, the failure of **reduction="none"** makes MRR impossible to calculate.

**Comment by Zhudongsheng75** [comment]
Sorry, I found the problem, you rewrote a LlamaForCausalLM.
